src/renderer/routes/routes.injectable.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { computed } from "mobx";
+import currentlyInClusterFrameInjectable from "./currently-in-cluster-frame.injectable";
+import allRoutesInjectable from "./all-routes.injectable";
+import { matches } from "lodash/fp";
+
+const routesInjectable = getInjectable({
+  id: "routes",
+
+  instantiate: (di) => {
+    const allRoutes = di.inject(allRoutesInjectable);
+
+    const currentlyInClusterFrame = di.inject(
+      currentlyInClusterFrameInjectable,
+    );
+
+    return computed(() =>
+      allRoutes
+        .get()
+        .filter(matches({ clusterFrame: currentlyInClusterFrame }))
+        .filter((route) => {
+          if (route.isEnabled) {
```

Comment:
I thought that this was required, so why the defensive code here?

Reply:
I don't understand.

Reply:
The interface `FrontEndRoute` has a field `isEnabled` of type `IComputedValue<boolean>` which is a required field (not an optional one) so this sort of check shouldn't be needed.

Reply:
You are right. Changing it tomorrow. Background: it wasn't required before, we changed our mind on that.

Reply:
Fixed it already.